kibana.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "security",
```

Comment:
What is different between this id and `PLUGIN_ID` mentioned inside `common/index.ts` d

Reply:
Good catch! it doesn't matter much, but better to make it consistent, changed it to `opendistroSecurity` in `kibana.json`